### PR TITLE
Solved Several Issues (Thank you in many languages, background change on hover, show info on hover)

### PR DESCRIPTION
**Pull request by Elgayar**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

Added several languages in thank you section and randomizing them by
implementing a shuffling function! Solves issue #947

Changed the background colors of contributors, mentors, and students
sections on hover to eyecatching colors! Solves issue #946

Made Show More Information in Students Section appear on hover instead
of clicking! Solves issue #948

Live Link: https://elgayar.github.io/gci16.fossasia.org/

### CODE REVIEW

**Comment by Elgayar** [comment]
Note: Some of my previous commits aren't appearing currently on my live link because they were done on a different branch, however, all the issues mentioned in the PR are working like charm on my live link! I'll merge all of them together soon.

https://twitter.com/MostafaaGayar/status/820460038041042945

File: javascripts/main.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
 // Add your function at the end. First function is of highest priority.
 var contributorsList = [];
+var thankList = ["Thanks", "Shukran", "Merci", "Sukria", "Danke", "Grazie", "Kamsa Hamnida", "Spasibo", "Obrigado"];
```
Comment (abishekvashok) [inline javascripts/main.js:3]: Can we have a Jekyll file for this so that students can add new languages in ease without having to modify the js files!

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: javascripts/main.js (hunk 1 above)
Comment (mariobehling) [reply javascripts/main.js:3]: @Abhi2424shek Please open a new issue for this.

**Comment by mariobehling** [comment]
@Elgayar Oh, I realized it too late: The "Contributors" section does not show the pics anymore. Please help to fix it. Thanks!

**Comment by mariobehling** [comment]
"Latest Issues" also not displaying.

**Comment by Elgayar** [comment]
They are all working to me. @mariobehling

**Pull request merged**